### ISSUE TITLE
Issues with showing which theme is current.

### ISSUE THREAD

**Issue by psych** [linuxmint/Cinnamon#125]
In Cinnamon 1.1.3 I added the "Cinnamon Minty" theme, shown here [1]. When I select it, it renders beautifully, however the default theme of "Cinnamon" remains highlighted as the current theme in use even though it is not [2]. When the default theme is in use it remains correctly highlighted [3]. You can see the different shades of highlight (green for the new Mint one and a clear for the default). When switching to one of the Gnome-Shell themes included by default in Gnome 3.2 + MGSE nothing is highlighted [4].
1. http://www.omgubuntu.co.uk/2012/01/cinnamon-desktop-gets-first-custom-theme/
2. http://i.imgur.com/u5MUl.png (notice the bottom panel changed to new theme as did the highlight color, but it is still shown as default). 
3. http://i.imgur.com/MCBAl.png
4. http://i.imgur.com/MvXmQ.png

In short, the default theme is always highlighted among the CInnamon themes, even when it is not active. When a Gnome-Shell theme is used nothing is highlighted.
